kuma/wiki/models.py

```diff
@@ -1543,6 +1543,20 @@ def contributors(self):
     def zone_stack(self):
         return DocumentZoneStackJob().get(self.pk)
 
+    @cached_property
+    def default_zone_stack(self):
+        """
+        The zone stack of the default-language parent of this
+        non-default-language document, otherwise an empty zone
+        stack.
+        """
```

Comment:
I'm not sure if this should be a separate method, or combined with the ``zone_stack`` method.